File: resultVisualizer/src/app/model/resultConfig.ts
import { Build } from "./build";
import { DefaultScale } from "./defaultScale";

export class ResultConfig {

    set Build(build: Array<Build>){
        this._build = build;
    }

    get Build(){
        return this._build;
    } 


    get DefaultScale(){
        return this._defaultScale;
    }

    constructor(private _build: Array<Build>, private _defaultScale : Array<DefaultScale>){}
}

File: com.viatra.cps.benchmark.reports.processing/src/com/viatra/cps/benchmark/reports/processing/models/ResultConfig.java
package com.viatra.cps.benchmark.reports.processing.models;

import java.util.List;

import org.codehaus.jackson.annotate.JsonProperty;

public class ResultConfig {
	
	@JsonProperty("Scale")
	protected List<Scale> scales;
	
	@JsonProperty("Build")
	protected List<Build> builds;

	public List<Scale> getScales() {
		return scales;
	}

	public void setScales(List<Scale> scales) {
		this.scales = scales;
	}

	public List<Build> getBuilds() {
		return builds;
	}

	public void setBuilds(List<Build> builds) {
		this.builds = builds;
	}
	

}
